VimeoNetworking/Sources/NSError+Extensions.swift

```diff
@@ -147,9 +145,10 @@ public extension NSError
      
      - returns: An error with additional information in the user info dictionary
      */
-    func errorByAddingDomain(domain: String?, code: Int?, userInfo: [String: AnyObject]?) -> NSError
+    func errorByAddingDomain(domain: String?, code: Int?, userInfo: [String: Any]?) -> NSError
```

Comment:
Should `[String: Any]` be `[AnyHashable: Any]`? Seems like we're dealing with both. Maybe these casts can be eliminated if we go with one